lib/Model/Listing/Dao/QueryBuilderHelperTrait.php

```diff
@@ -39,9 +52,8 @@ protected function applyListingParametersToQueryBuilder(QueryBuilder $queryBuild
         $this->applyOrderByToQueryBuilder($queryBuilder);
         $this->applyLimitToQueryBuilder($queryBuilder);
 
-        $callback = $this->onCreateQueryBuilderCallback;
-        if (is_callable($callback)) {
-            $callback($queryBuilder);
+        foreach ($this->queryBuilderProcessors as $processor) {
```

Comment:
might be good to add a getter method and replace with

```suggestion
        foreach ($this->getQueryBuilderProcessors() as $processor) {
```

Reply:
I don't think it makes sense - the property is used inly inside the trait, so getter is excess. Also you can see how it was implemented before my changes.

Reply:
ok, sounds fine to me, thank you for the feedback